src/lasp_transmission_instrumentation.erl

```diff
@@ -57,53 +57,59 @@
 
 %% @doc Start and link to calling process.
 -spec start_link(list(term()))-> {ok, pid()} | ignore | {error, term()}.
-start_link(Type) ->
-    gen_server:start_link({global, {?MODULE, Type}}, ?MODULE, [Type], []).
+start_link(Entity) ->
+    gen_server:start_link({global, {?MODULE, Entity}}, ?MODULE, [Entity], []).
 
--spec log(term(), term(), node()) -> ok | error().
-log(Type, Term, Node) ->
-    gen_server:call({global, {?MODULE, Type}}, {log, Term, Node}, infinity).
+-spec log(term(), term(), term(), pos_integer(), node()) -> ok | error().
+log(Entity, Type, Payload, PeerCount, Node) ->
+    gen_server:call({global, {?MODULE, Entity}}, {log, Type, Payload, PeerCount, Node}, infinity).
 
 -spec start(term(), list(), pos_integer()) -> ok | error().
-start(Type, Filename, Clients) ->
-    gen_server:call({global, {?MODULE, Type}}, {start, Filename, Clients}, infinity).
+start(Entity, Filename, Clients) ->
+    gen_server:call({global, {?MODULE, Entity}}, {start, Filename, Clients}, infinity).
```

Comment:
First, start by removing Entity and Type, because those attributes are not required.

Then, you should be able to change it to `{local, ?MODULE}`.